util/MySqlMigrations/Migrations/20231214162533_GrantIdWithIndexes.cs

```diff
@@ -73,13 +72,7 @@ protected override void Up(MigrationBuilder migrationBuilder)
             .Annotation("MySql:CharSet", "utf8mb4")
             .OldAnnotation("MySql:CharSet", "utf8mb4");
 
-        migrationBuilder.AddColumn<int>(
-            name: "Id",
-            table: "Grant",
-            type: "int",
-            nullable: false,
-            defaultValue: 0)
-            .Annotation("MySql:ValueGenerationStrategy", MySqlValueGenerationStrategy.IdentityColumn);
+        migrationBuilder.Sql("ALTER TABLE `Grant` ADD COLUMN `Id` INT AUTO_INCREMENT UNIQUE;");
```

Comment:
ℹ️ Note that I make this a unique index for the moment, then add the named primary key. I couldn't find a way to one-line the primary key creation _with a name_ -- it could be created but not named.